File: examples/py/coinone-markets.py
# -*- coding: utf-8 -*-

import os
import sys
from pprint import pprint

root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
sys.path.append(root + '/python')

import ccxt  # noqa: E402

exchange = ccxt.coinone({
    'enableRateLimit': True,
    # 'verbose': True,  # uncomment for verbose output
})

markets = exchange.load_markets()
pprint(markets)
print('\n', exchange.name, 'supports', len(markets), 'pairs')

File: examples/php/coinone-markets.php
<?php

$root = dirname (dirname (dirname (__FILE__)));

include $root . '/ccxt.php';

$exchange = new \ccxt\coinone (array (
    'enableRateLimit' => true,
    // 'verbose' => true, // uncomment for verbose output
));

$markets = $exchange->load_markets();

var_dump ($markets);
echo "\n" . $exchange->name . " supports " . count($markets) . " pairs\n";

?>
